site/en/docs/devtools/open/index.md

```diff
@@ -4,68 +4,125 @@ title: "Open Chrome DevTools"
 
-## Open the last panel you had open {: #last }
+DevTools opens the **Elements** panel and selects the element in the DOM tree. In the **Styles** pane, you can see CSS rules applied to the selected element.
 
-Press <kbd>Command</kbd>+<kbd>Option</kbd>+<kbd>I</kbd> (Mac) or <kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>.
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WTJEhyPfarXQqli3LMwW.png", alt="An element selected in the Elements panel.", width="800", height="469" %}
 
-## Open DevTools from Chrome's main menu {: #chrome }
+### Open the last panel you used from Chrome's main menu {: #last }
 
-Click **Customize and control Google Chrome**
-{% Img src="image/admin/t7rsoT0IGcnwvW76Xj32.png", alt="Customize and control Google Chrome", width="6", height="26" %} and
-then select **More Tools** > **Developer Tools**.
+To open the last DevTools panel, click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N7wEDmtW9lnrSxPRupMa.svg", alt="Three-dot menu.", width="22", height="22" %} button to the right of the address bar and select **More Tools** > **Developer Tools**.
 
-{% Img src="image/admin/0bZRHFrsZGxpTAWhYCg6.png", alt="Opening DevTools from Chrome's main menu.", width="800", height="492" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/bkh79zEFaByczisr3lD5.png", alt="The Developer Tools option selected from the three-dot menu.", width="800", height="516" %}
 
-**Figure 2**. Opening DevTools from Chrome's main menu
+Alternatively, you can open the last panel with a shortcut. See the next section to learn more.
+
+## Open panels with shortcuts: Elements, Console, or your last panel  {: #shortcuts }
+
+If you prefer keyboard, press a shortcut in Chrome depending on your operating system:
+
+<table>
+<thead>
+  <tr>
+    <th>OS</th>
+    <th>Elements</th>
+    <th>Console</th>
+    <th>Your last panel</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Windows or Linux</td>
+    <td><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>C</strong></td>
+    <td><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>J</strong></td>
+    <td><kbd>F12</kbd></br><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>I</strong></td>
+  </tr>
+  <tr>
+    <td>Mac</td>
+    <td><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>C</strong></td>
+    <td><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>J</strong></td>
+    <td><kbd>Fn</kbd> + <kbd>F12</kbd></br><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>I</strong></td>
+  </tr>
+</tbody>
+</table>
+
+Here's an easy way to memorize the shortcuts:
+
+- **C** stands for CSS.
+- **J** for JavaScript.
+- **I** designates your choice.
+
+The **C** shortcut opens the **Elements** panel in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/7s3JQLXmIQmQa4CFXaNv.png", alt="Inspect.", width="21", height="20" %} inspector mode. This mode shows you helpful tooltips when you hover over elements on a page.
+
```

Comment:
Add a line to tell users that they can click on any elements to inspect its styles in the Styles pane.